app/ante_no_seq.go

```diff
@@ -0,0 +1,67 @@
+//go:build excludeIncrement
```

Comment:
This ante handler only build with the excludeIncrement build flag, which will only be used in E2E